services/minespace-web/src/components/common/CoreTable.tsx

```diff
@@ -64,15 +64,21 @@ const CoreTable = <T,>(props: CoreTableProps<T>) => {
             placement="right"
             mouseEnterDelay={1}
           >
-            <MinusSquareFilled className="icon-lg--lightgrey" />
+            <MinusSquareFilled
+              style={{ fontSize: "24px", color: "#ded9d9" }}
+              className="icon-lg--lightgrey"
+            />
           </Tooltip>
         ) : (
           <Tooltip
             title={`Click to show ${recordDescription}.`}
             placement="right"
             mouseEnterDelay={1}
           >
-            <PlusSquareFilled className="icon-lg--lightgrey" />
+            <PlusSquareFilled
+              style={{ fontSize: "24px", color: "#ded9d9" }}
```

Comment:
I noticed you're using inline styles in a few areas.  For me, it's not a hill to die on, but I much prefer creating classes in the css files wherever possible.